salsa-2022-tests/tests/tracked_method.rs

```diff
@@ -0,0 +1,44 @@
+//! Test that a `tracked` fn on a `salsa::input`
+//! compiles and executes successfully.
+#![allow(warnings)]
+
+#[salsa::jar(db = Db)]
+struct Jar(MyInput, MyInput_tracked_fn, MyInput_tracked_fn_ref);
+
+trait Db: salsa::DbWithJar<Jar> {}
+
+#[salsa::input(jar = Jar)]
+struct MyInput {
+    field: u32,
+}
+
+#[salsa::tracked(jar = Jar)]
```

Comment:
```suggestion
#[salsa::tracked]
```

This is the default, presumably?

Reply:
Yes I was just copying the other tests which appear to all specify the jar type for some reason.

Reply:
I've removed it anyway.